### PR TITLE
Adds MD5 to file metadata.

### LINKED ISSUE #110: Add md5 checksums into contentMetadata from Google
This is to ensure the checksums get verified in assembly WF in the checksum-compute step.   This robot will recompute checksums and confirm they match with the values you provided.

Note: Confirm that is how the robot works in assemblyWF

### PR DESCRIPTION
**Pull request by justinlittman**
closes #110

## Why was this change made?
So that the MD5 that is provided with Google is passed to the SDR Deposit API so that it can be checked.

## Was the documentation (README, DevOpsDocs, wiki) updated?
No.

### CODE REVIEW

File: app/services/file_metadata_builder.rb [hunk 1]
```diff
@@ -46,5 +50,20 @@ def build
 
   private
 
-  attr_reader :file_paths
+  attr_reader :directory
+
+  def checksums
```
Comment (mjgiarlo) [inline app/services/file_metadata_builder.rb]: I'm curious why this logic lives here and not in the `MD5` class.

Comment (justinlittman) [reply app/services/file_metadata_builder.rb]: It should, but could not figure out how to do this given that `<operation>.for` is a class method. Open to suggestions.

Comment (mjgiarlo) [reply app/services/file_metadata_builder.rb]: @justinlittman OK, cool. wanna pair?

File: app/services/download_directory.rb [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 # This represents a directory of downloaded files
 class DownloadDirectory
+  attr_reader :directory
```
Comment (jcoyne) [inline app/services/download_directory.rb]: why does this need to be public?

Comment (jcoyne) [reply app/services/download_directory.rb]: NM, i see.

Comment (jcoyne) [reply app/services/download_directory.rb]: How about creating a `checksum_file` or `checksums` method in here instead?

Comment (justinlittman) [reply app/services/download_directory.rb]: Added checksums method here.

File: spec/services/download_directory_spec.rb [hunk 3]
```diff
@@ -3,29 +3,40 @@
...
   describe '#files' do
     subject(:files) { instance.files }
 
-    let(:filenames) do
-      ['downloads/36105006348168/00000800.jp2',
-       'downloads/36105006348168/00000751.html',
-       'downloads/36105006348168/00000917.html',
-       'downloads/36105006348168/00000653.jp2']
-    end
+    # let(:filenames) do
```
Comment (jcoyne) [inline spec/services/download_directory_spec.rb]: cruft?

Comment (justinlittman) [reply spec/services/download_directory_spec.rb]: Decrufted. Sorry about that.

**Pull request merged**